deploy/env_vars/base.yml

```diff
@@ -50,6 +54,20 @@ sg_list:
         to_port: 80
         group_name: "{{ lb_sg_name }}"
         group_desc: "{{ lb_sg_description }}"
+    rules_egress:
+      - proto: tcp
+        from_port: 80
+        to_port: 80
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        from_port: 443
+        to_port: 443
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        from_port: 5432
+        to_port: 5432
+        group_name: "{{ db_sg_name }}"
+        group_desc: "{{ db_sg_description }}"
```

Comment:
The section above sets rules for incoming and outgoing network traffic for the load balancer, webservers, and database.